internal/log/olog/proxy.go

```diff
@@ -0,0 +1,83 @@
+package olog
+
+import "io"
+
+// SetOutput is a proxy for log.SetOutput.
```

Comment:
It seems the only parts we use from this currently is "Printf()", can we YAGNI this and just remove everything here (with a comment maybe) and just keep "Prinf()" for now and add as needed?

Reply:
Yeah I wanted to implement all methods because I very often see APIs defining interface for the standard library logger struct in various detail, some only have `Print` functions, others include `Panic` and some actually have all of the methods.

This API will not change anytime soon, I am pretty sure. So we could just leave that there. Or a compromise: just keep all the logging methods and leave setters/getters, except `Writer` that is a useful one.